docs/Reference.md

```diff
@@ -192,7 +192,7 @@ Using the markdown for either users, roles, or channels will mention the target(
 ###### Image Base Url
 
 ```
-https://cdn.discordapp.com/
+https://cdn.discord.com/
```

Comment:
I think this is not gonna change

Reply:
It's still `cdn.discordapp.com` at the moment, and a DNS lookup for `cdn.discord.com` doesn't currently return any results.

Reply:
it's gonna stay on discordapp.com